server/channels/app/authorization.go

```diff
@@ -287,31 +287,34 @@ func (a *App) HasPermissionToTeam(rctx request.CTX, askingUserId string, teamID
 	return a.HasPermissionTo(askingUserId, permission)
 }
 
-func (a *App) HasPermissionToChannel(rctx request.CTX, askingUserId string, channelID string, permission *model.Permission) bool {
+func (a *App) HasPermissionToChannel(rctx request.CTX, askingUserId string, channelID string, permission *model.Permission) (bool, bool) {
```

Comment:
The two bool return values are confusing. Let's add a comment explaining what they do.